coderd/licenses.go

```diff
@@ -0,0 +1,24 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+
+	"github.com/coder/coder/coderd/httpapi"
+	"github.com/coder/coder/codersdk"
+)
+
+func Licenses() http.Handler {
+	r := chi.NewRouter()
+	r.NotFound(licensesUnsupported)
+	return r
+}
+
+func licensesUnsupported(rw http.ResponseWriter, _ *http.Request) {
```

Comment:
This says `licensesUnsupported`, but doesn't mention licensing in the response. I'd suggest making this `enterpriseOnly` or something instead.